tests/ci/build-release.yaml

```diff
@@ -28,7 +28,13 @@ steps:
       echo "Expecting value for _GITHUB_API_TOKEN"
       exit 1
     fi
-    git clone --depth 1 -b dev https://$_GITHUB_API_TOKEN@github.com/vectorizedio/vtools vtools
+
+    git clone --depth 1 https://$_GITHUB_API_TOKEN@github.com/vectorizedio/vtools vtools
```

Comment:
From our convo on Slack, let's move this to a script so it can be reused.

Reply:
for GCB, we define all bash scripts in line, though we could move them to out of line.

Reply:
Send the portrait